packages/website/src/components/RulesTable/index.tsx

```diff
@@ -191,29 +217,69 @@ export default function RulesTable({
           <RuleFilterCheckBox
             mode={filters.fixable}
             setMode={(newMode): void => changeFilter('fixable', newMode)}
-            label="🔧 fixable"
+            label={`${FIXABLE_EMOJI} fixable`}
           />
           <RuleFilterCheckBox
             mode={filters.suggestions}
             setMode={(newMode): void => changeFilter('suggestions', newMode)}
-            label="💡 has suggestions"
+            label={`${SUGGESTIONS_EMOJI} has suggestions`}
           />
           <RuleFilterCheckBox
             mode={filters.typeInformation}
             setMode={(newMode): void =>
               changeFilter('typeInformation', newMode)
             }
-            label="💭 requires type information"
+            label={`${TYPE_INFORMATION_EMOJI} requires type information`}
+          />
+          <RuleFilterCheckBox
+            mode={filters.extension}
+            setMode={(newMode): void => changeFilter('extension', newMode)}
+            label={`${EXTENSION_RULE_EMOJI} extension rule`}
+          />
+          <RuleFilterCheckBox
+            mode={filters.formatting}
+            setMode={(newMode): void => changeFilter('formatting', newMode)}
+            label={`${FORMATTING_RULE_EMOJI} formatting rule`}
+          />
+          <RuleFilterCheckBox
+            mode={filters.deprecated}
+            setMode={(newMode): void => changeFilter('deprecated', newMode)}
+            label={`${DEPRECATED_RULE_EMOJI} deprecated rule`}
```

Comment:
![Screenshot of the rule filters wrapping the last one, deprecated, to the next line](https://github.com/typescript-eslint/typescript-eslint/assets/3335181/95a6e43f-5a16-4774-bf28-f795e8c5697e)

It's going to bug me that on my monitor, these wrap to a next line... maybe one of the titles can be shrunk by a word?

Reply:
that seems arbitrary, but i'll get rid of "rule" in the three places that it appears

Reply:
#7992 -> #7993